src/mon/MonClient.cc

```diff
@@ -572,12 +572,11 @@ int MonClient::authenticate(double timeout)
   if (!_opened())
     _reopen_session();
 
-  auto until = ceph::mono_clock::now();
```

Comment:
@ideepika could you please put more details on this change?  for instance, what is the symptom of the issue? what is its impact? why does this change address the issue?

Reply:
@tchaikov will do, sometimes monclient hangs forever holding exclusive lock in my local environment. But on testing without this things work fine now, needs further investigation, converting to draft